### DISCUSSION TITLE
More database notifications to users but One Notification to Microsoft Teams channel

### DISCUSSION THREAD

**Discussion by francesco-loreti** [excid3/noticed#217] [Q&A] [answered]
Hi,
I've created a Notification and send a Notify to each user on commit. 
After, I have added a delivery channel on Microsoft Teams into same Notification. 
Now on that channel I receive a number of notifications equal to the number of users.
Is it possible to have only one notification sent on the Teams channel without having to recreate a new Notification?

Thanks!!!

**Comment by excid3** [comment, marked as answer]
You'll want two separate notifications then. One for individual recipients and one for the shared recipient (Teams)

**Reply by francesco-loreti** [reply]
I was hoping I didn't have to :-(
